flash_tutorials/electricity_forecasting/electricity_forecasting.py

```diff
@@ -155,7 +155,7 @@ def preprocess(df: pd.DataFrame, frequency: str = "1H") -> pd.DataFrame:
 
 # %%
 trainer = flash.Trainer(
-    max_epochs=3,
+    max_epochs=1,
```

Comment:
Mind adding a reason why only 1?

Reply:
Oh, this was during debugging as the CI was failing. Timeout for each cell is 300 seconds, and I was trying to reduce the epochs to see if it fixes it. I'll move it back to `3`.

```suggestion
    max_epochs=3,
```